scripts/performance-helper.psm1

```diff
@@ -566,6 +580,8 @@ class ThroughputTestPublishResult {
     }
 }
 
+$ThroughputUpRegressionThreshold = -5.0
```

Comment:
This should be in the json, per test, to say what the "failure threshold" is. For throughput up, 5% is good. For RPS, 50% seems to be required.